### DISCUSSION TITLE
Sponsor/patreon link? Also, thanks for removing `ctrl-r` hook by default (iirc)

### DISCUSSION THREAD

**Discussion by jakswa** [zellij-org/zellij#1003] [Ideas]
I just updated from my old version (0.14.0) to 0.24.0 and was inordinately excited to find that I could use `ctrl-r` (reverse command search in my terminal) without triggering a zellij hook. To be fair I was fine realizing I needed to lock zellij (`ctrl-g`) before... but finding that it works with my flow easier is awesome! :sparkles:

I was so happy that I scanned the github quickly for a patreon/donation link to buy the dev(s) a coffee/beer, but came up empty handed. So big thanks, and also lmk if I missed a donation link somewhere.

**Comment by imsnif** [comment]
Hey, I'm really glad to hear this change made you so happy. Wow, the upgrade from 0.14.0 to 0.24.0 includes quite some changes! I hope you like them all.

While not officially linked to the project (yet?) - I'm always happy for sponsorships if you are so inclined (incidentally, I'm also the one who initiated this change): https://github.com/sponsors/imsnif

Cheers!

**Reply by jakswa** [reply]
awesome! thanks and done :white_check_mark:

![image](https://user-images.githubusercontent.com/137793/148831992-db248da9-8451-4ac1-ba87-99fc5d4e5ec2.png)

**Reply by imsnif** [reply]
Very much appreciated, thank you!
